### DISCUSSION TITLE
Extend IDSA Rulebook / Introduction

### DISCUSSION THREAD

**Discussion by anilturkmayali** [International-Data-Spaces-Association/Personal-Data-and-IDS#3] [Ideas]
**What could be elaborated?** 
The introduction could provide a more detailed overview of the importance of personal data within the IDS framework.

**Action Steps:**
- Highlight the significance of personal data in the current digital landscape.
- Emphasize the challenges and opportunities of integrating personal data within IDS.
- Provide a brief overview of the working group's goals related to personal data.

**Comment by AlexTourski** [comment]
Dear Anil, 
This is a general comments to all RAM/Rulebook discussions entries:
if (IF) the use cases are an appropriate language to go one step down from the Manifesto/Declaration, then the RAM/Rulebook discussions have to follow such use cases, not precede them. 
E.g. with regard to "Long-term preservation":
if "long-term is 30, we need one RAM, but if it requires 500 years, it is a completely different RAM. 

To sum it up, I suggest the 3-layered approach in getting to RAM:
1. Declaration
2. Use cases
3. RAM

Does it make sense?

Best Regards,
Alex Tourski
Founder
postplatforms.org
= = = = = = = = = 
mob: +31645388324
email: alex.tourski@postplatforms.org

**Reply by anilturkmayali** [reply]
Thank you for the comment @AlexTourski. 
As far as I understood from the example you gave I think you mean that once we decide to update any specific session in IDS RAM and/or IDSA Rulebook we need to take use-cases into consideration and verify the update's validity via thinking it in that use-case's context, right?

**Reply by AlexTourski** [reply]
Dear @anilturkmayali, 
Yes, something like that. 
Use cases are "guidelines" for engineers to check "whether we go to the right direction, defined by the strategists"? It's the language between strategists (like me) and architects. Both in case if IDS and Solid we lack strategic guiding, and use cases will help us. 

Last week I have seen a wonderful case at the Europeana Data Space conference: After hours of talking about Data Space in very vague language a lady stood up and asked a very simple questions: "Guys, do you have use cases to explain what are you doing as I do not understand anything from this discussion on Data Space?"
For 2 minutes it was a silence, which spoke for itself.
